build.gradle

```diff
@@ -85,6 +85,7 @@ dependencies {
     commonImplementation 'org.apache.maven:maven-artifact:3.6.0'
     commonImplementation 'org.apache.httpcomponents:httpclient:4.3.3'
     commonImplementation 'net.minecraftforge:srgutils:0.2.12'
+    commonImplementation 'codechicken:DiffPatchGradle:1.0.0.+'
```

Comment:
I would prefer if we pulled in the library itself, and implemented the tasks themselves instead of going the indirect route.